causalpy/experiments/diff_in_diff.py

```diff
@@ -102,6 +104,21 @@ def __init__(
         self.y, self.X = np.asarray(y), np.asarray(X)
         self.outcome_variable_name = y.design_info.column_names[0]
 
+        # turn into xarray.DataArray's
+        self.X = xr.DataArray(
+            self.X,
+            dims=["obs_ind", "coeffs"],
```

Comment:
dims of X as coeffs? Should surely be covariates... coeffs is the output, and while they should be 1:1 naming is a little misleading no?

Reply:
My motivation for calling it `coeffs` was the the use case of extracting the inferred coefficient values. I agree it does make more sense to label these as covariates when you are building the model and its dims. I might put together an issue to describe this - it could be a good refactor, but I'd say it's best dealt with separately.